UnityProject/Assets/Scripts/Objects/Doors/DoorAnimatorV2.cs

```diff
@@ -311,12 +381,12 @@ public void RemovePanelOverlay()
 		}
 
 		/// <summary>
-		/// Used to call coroutines from outside monobehaviors
+		/// Used to call UniTasks from outside monobehaviors
 		/// </summary>
 		/// <param name="anim"></param>
-		public void RequestAnimation(IEnumerator anim)
+		public void RequestAnimation(UniTaskVoid anim)
```

Comment:
technically, since this isn't a coroutine anymore, this pattern shouldn't be needed